### PR TITLE
Add target-utilization option for `service create` and `service update`

### PR DESCRIPTION
**Pull request by MIBc**
Fixes: #https://github.com/knative/client/issues/768

* Support setting "autoscaling.knative.dev/targetUtilizationPercentage" annotation.

Release Note:

- 🎁 New feature
Add "--target-utilization" to manage "autoscaling.knative.dev/targetUtilizationPercentage" annotation.

### CODE REVIEW

**Comment by maximilien** [comment]
/ok-to-test

You might first need to resolve conflict: `test/e2e/service_options_test.go`

**Comment by rhuss** [comment]
The test is failing because we have switched back to async delete by default and haven't adapted the e2e tests to add a `--wait` to the delete operations.

@duglin you asked for an evidence why sync deletion is important in e2e tests. The [it failure here]( https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/knative_client/788/pull-knative-client-integration-tests-latest-release/1248132495577190400) is one of it.

File: pkg/kn/commands/service/describe_test.go [hunk 1]
```diff
@@ -253,13 +253,13 @@ func TestServiceDescribeScaling(t *testing.T) {
 
 	for _, data := range []struct {
 		minScale, maxScale, limit, target string
-		scaleOut                          string
+		scaleOut, utilization                           string
```
Comment (mattmoor) [inline pkg/kn/commands/service/describe_test.go:256]: Format Go code:
```suggestion
		scaleOut, utilization             string
```

File: pkg/serving/config_changes_test.go [hunk 2]
```diff
@@ -723,9 +726,10 @@ func checkContainerConcurrency(t *testing.T, template *servingv1.RevisionTemplat
 	}
 }
 
-func updateConcurrencyConfiguration(template *servingv1.RevisionTemplateSpec, minScale int, maxScale int, target int, limit int) {
+func updateConcurrencyConfiguration(template *servingv1.RevisionTemplateSpec, minScale, maxScale, target ,limit, utilization int) {
```
Comment (mattmoor) [inline pkg/serving/config_changes_test.go:729]: Format Go code:
```suggestion
func updateConcurrencyConfiguration(template *servingv1.RevisionTemplateSpec, minScale, maxScale, target, limit, utilization int) {
```

**Pull request merged**